Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -478,6 +485,22 @@ private void TrySetParam(DB.Element elem, BuiltInParameter bip, double value, st
       }
     }
 
+    /// <summary>
+    /// Queries a Revit Document for phases by the given name.
+    /// </summary>
+    /// <param name="document"></param>
+    /// <param name="phaseName">The name of the Phase</param>
+    /// <returns>the phase which has the same name. null if none or multiple phases were found.</returns>
+    private Element GetRevitPhase(DB.Document document, string phaseName )
+    {
+      var phases = new FilteredElementCollector(document).OfCategory(BuiltInCategory.OST_Phases)
+        .Where(ph => ph.Name.Equals(phaseName));
+      if (phases.Count() == 1) return phases.First();
```

Comment:
Can simplify with `return phases.FirstOrDefault()`

Reply:
This isn't applicable anymore